### PR TITLE
improve beginner friendliness

### PR DESCRIPTION
**Pull request by Clockworkx**

### CODE REVIEW

File: docs/setup/Connecting.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Connecting to the Server
+
+## Launcher (Recommended)
+1. Download [MS2 Launcher](https://github.com/Clockworkx/MS2-Launcher/releases/download/v1.0/MS2.Launcher.v1.0.1.7z).
+2. Extract the launcher to a folder.
+3. Make a shortcut of MS2Launcher.Exe (rightclick->Send to->Desktop)
+4. Start the Launcher.
+5. You can configure your IPs using the disk-pen symbol on the bottom-right of the launcher. Change your IP and Port and click on the save icon.
+6. Select whether you want to play Local (using the server on your PC) or Online (the remote server configured in step 5).
+7. Click Start. On first lauunch the Launcher will ask you to select your MapleStory2.exe.
```
Comment (AngeloTadeucci) [inline docs/setup/Connecting.md:18]:
```suggestion
7. Click Start. On first launch the Launcher will ask you to select your MapleStory2.exe location.
```

File: docs/setup/Connecting.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## Manual
+
+If your server is running on your PC you can proceed with Connecting to Localhost. If you are playing on a server that is NOT on your computer, go to the Connecting to Remote Server section.
+
+## Connecting to Localhost
+
+1. If you have NOT done any changes to your server port, you are ready to play: Doubleclick the MapleStory2.exe which is located in Your_MapleStoryClient_Installation_Path\appdata\MapleStory2.exe.
+   
+2. You can select any server. The id and password that you enter will be used to CREATE an account, once you click on login. So make sure to remember your data. 
+
```
Comment (AngeloTadeucci) [inline docs/setup/Connecting.md:32]: This really isn't step two, the reader have not even opened the client

File: docs/setup/Connecting.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 7
+---
+
+# Connecting to the Server
+
+## Launcher (Recommended)
+1. Download [MS2 Launcher](https://github.com/Clockworkx/MS2-Launcher/releases/download/v1.0/MS2.Launcher.v1.0.1.7z).
+2. Extract the launcher to a folder.
+3. Make a shortcut of MS2Launcher.Exe (rightclick->Send to->Desktop)
```
Comment (AngeloTadeucci) [inline docs/setup/Connecting.md:14]:
```suggestion
3. Make a shortcut of **MS2Launcher.exe* (Right click -> Send to -> Desktop)
```

File: docs/setup/Connecting.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+* Note: You can see the online status of the server in the top-right corner of the corresponding server.
+
+## Manual
+
+If your server is running on your PC you can proceed with Connecting to Localhost. If you are playing on a server that is NOT on your computer, go to the Connecting to Remote Server section.
+
+## Connecting to Localhost
+
+1. If you have NOT done any changes to your server port, you are ready to play: Doubleclick the MapleStory2.exe which is located in Your_MapleStoryClient_Installation_Path\appdata\MapleStory2.exe.
```
Comment (AngeloTadeucci) [inline docs/setup/Connecting.md:29]:
```suggestion
1. If you have NOT done any changes to your server port, you are ready to play: Double click the **MapleStory2.exe** which is located in `Your_MapleStoryClient_Installation_Path\appdata\MapleStory2.exe`.
```

File: docs/setup/Connecting.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+   
+2. You can select any server. The id and password that you enter will be used to CREATE an account, once you click on login. So make sure to remember your data. 
+
+3. You can rightclick -> Send to -> Desktop (create shortcut), so you can start Maple Story from your Desktop, next time.
```
Comment (AngeloTadeucci) [inline docs/setup/Connecting.md:33]:
```suggestion
3. You can right click -> Send to -> Desktop (create shortcut) so you can start MapleStory2 more easily.
```

File: docs/setup/Connecting.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+4. If you have NOT done any changes to your server ports, you are ready to play. Enjoy.
```
Comment (AngeloTadeucci) [inline docs/setup/Connecting.md:35]: I think this can be removed, already mentioned on step 1

File: docs/setup/Connecting.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+4. If you have NOT done any changes to your server ports, you are ready to play. Enjoy. 
+
+5. If you made changes to your server ports, you need to create a shortcut as in Step 3.
+   
+6. Go to the properties of the shortcut and in the Target field you add your ip and LOGIN port (can be seen in your .env file) like so:
+    ...appdata\MapleStory2.exe" --ip=127.0.0.1 --port=20001 --nxapp=nxl --lc=EN
+
+    Click OK and you can now start the game via your SHORTCUT. 
+
+* Note: You can also leave both id and password empty, which will create an account with no id and no password. Everyone will be able to log into this account.
```
Comment (AngeloTadeucci) [inline docs/setup/Connecting.md:44]:
```suggestion
6. Go to the properties of the shortcut and in the target field, add your ip and **LOGIN** port (can be found in your .env file) like so: `...appdata\MapleStory2.exe" --ip=127.0.0.1 --port=20001 --nxapp=nxl --lc=EN`

7. Click OK and you can now start the game via your SHORTCUT. 

* Note: When you reach the login screen, you can also leave both id and password empty, which will create an account with no id and no password. Everyone will be able to log into this account.
Accounts are auto-generated, so you can use whatever username and password you want.
```

File: docs/setup/running.md [hunk 8]
```diff
@@ -4,7 +4,12 @@ sidebar_position: 5
 
 # Running the Emulator
 
-## Visual Studio Code Guide
+## Visual Studio 2022 (Recommended)
+
+1. Click RUN or use the shortcut (F5). This will start your server.
+2. you can now proceed with the Connection part of the docs.
```
Comment (AngeloTadeucci) [inline docs/setup/running.md:10]:
```suggestion
2. You can now proceed with the Connection part of the docs.
```

File: docs/setup/Connecting.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 7
+---
+
+# Connecting to the Server
+
+You can choose between starting the client manually or using a launcher to do the work for you.
+If you do not plan to switch between different servers (local and online), you might choose the manual method, which will guide you in creating a shortcut for the client that connects to the server.
+Alternatively, you can use a launcher that does all the work for you. In the end, starting the client from a shortcut might be faster than first starting the launcher, but the launcher allows for easy configuration, if needed. Your choice.
```
Comment (AngeloTadeucci) [inline docs/setup/Connecting.md:9]:
```suggestion
You can choose between starting the client manually or using a launcher to do the work for you.
If you do not plan to switch between different servers (local and online), you might wanna choose the manual method, which will guide you in creating a shortcut for the client that connects to the server.
Alternatively, you can use a launcher that does all the work for you. In the end, starting the client from a shortcut might be faster than first starting the launcher, but the launcher allows for easy configuration, if needed. Your choice.
```

File: docs/setup/Connecting.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+    ...appdata\MapleStory2.exe" --ip=127.0.0.1 --port=20001 --nxapp=nxl --lc=EN
+
+    Click OK and you can now start the game via your SHORTCUT. 
+
+* Note: You can also leave both id and password empty, which will create an account with no id and no password. Everyone will be able to log into this account.
+
+### Connecting to Remote Server
+1. If you want to join a server running on another computer, refer to step 5 and 6 of Connecting to Localhost. 
+2. In the --ip= parameter you enter the ip of the computer the maple server is running on.
+3. In the --port= parameter you enter the LOGIN server port of the maple server that you want to connect to.
```
Comment (AngeloTadeucci) [inline docs/setup/Connecting.md:49]:
```suggestion
### Connecting to Remote Server
1. Follow the steps in Connecting to Localhost.
2. Redo step 6 following the details below.
3. In the `--ip=` parameter you enter the ip of the computer on which MapleServer is running on.
4. In the `--port=` parameter you enter the **LOGIN** server port of the MapleServer that you want to connect to. 
```

**Review by AngeloTadeucci — CHANGES_REQUESTED** [review]

File: docs/setup/prerequisites.md [hunk 11]
```diff
@@ -4,12 +4,25 @@ sidebar_position: 1
...
 # Prerequisites
 
-1. Visual Studio 2022 or Visual Studio Code. Visual Studio Code is recommended. If you choose Visual Studio instead, make sure you get the community edition and use version 2022. **2019 will not work, only 2022.**
+1. Download Visual Studio 2022 Community Edition
+   
+   - [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) (RECOMMENDED)
 
-    - [Visual Studio Code](https://code.visualstudio.com/?wt.mc_id=vscom_downloads)
-    - [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)
+If you are a beginner, you want to use Visual Studio 2022 NOT Visual Studio CODE for the entirety of the setup. Remember, if you decide for Visual Studio 2022, you only need to do the Visual Studio 2022 parts of the tutorial and you can skip the (NOT Visual Studio CODE sections.
```
Comment (AngeloTadeucci) [inline docs/setup/prerequisites.md:11]:
```suggestion
If you are a beginner, you want to use Visual Studio 2022 NOT Visual Studio CODE for the entirety of the setup. Remember, if you decide for Visual Studio 2022, you only need to do the Visual Studio 2022 parts of the tutorial and you can skip the other sections.
```

File: docs/setup/prerequisites.md [hunk 12]
```diff
@@ -4,12 +4,25 @@ sidebar_position: 1
...
+    **Versions below 2022 will NOT work**
+
+You can skip to step 2 if you do not plan to use another environment than Visual Studio 2022.
+
+If you know what you are doing, we also have instructions how to setup the server using Visual Studio CODE but it is NOT recommended for beginners.
+You can download it here:
+
+- [Visual Studio Code](https://code.visualstudio.com/?wt.mc_id=vscom_downloads)
+
+In case you prefer a Visual Studio alternative, you can of course also use Rider. The largest difference between Rider and Visual Studio will be, perhaps, different namings for the menu options, but you should be able to follow along the Visual Studio Setup analogously.
```
Comment (AngeloTadeucci) [inline docs/setup/prerequisites.md:23]: Rider is paid so not worth saying its a good alternative

**Review by AngeloTadeucci — CHANGES_REQUESTED** [review]

File: docs/setup/running.md [hunk 13]
```diff
@@ -1,13 +1,13 @@
...
+sidebar_position: 6
 ---
 
 # Running the Emulator
 
 ## Visual Studio 2022 (Recommended)
 
 1. Click RUN or use the shortcut (F5). This will start your server.
-2. you can now proceed with the Connection part of the docs.
+2. you can now proceed with the Web Server part of the docs or skip directly to Connecting to the Server if you do not want to enable UGC content.
```
Comment (AngeloTadeucci) [inline docs/setup/running.md:10]: You capitalized and mention the same on other sections

File: docs/setup/Connecting.md (hunk 2 above)
Comment (Clockworkx) [reply docs/setup/Connecting.md:32]: You didn't see the second part of step 1, I think
1. If you have NOT done any changes to your server port, you are ready to play: Doubleclick the MapleStory2.exe which is located in Your_MapleStoryClient_Installation_Path\appdata\MapleStory2.exe.

File: docs/setup/Connecting.md (hunk 4 above)
Comment (Clockworkx) [reply docs/setup/Connecting.md:29]: I wanted it to be ports not port, so I added your changes in a commit!

File: docs/setup/Connecting.md (hunk 5 above)
Comment (Clockworkx) [reply docs/setup/Connecting.md:33]: added in a commit.

File: docs/setup/Connecting.md (hunk 6 above)
Comment (Clockworkx) [reply docs/setup/Connecting.md:35]: agreed. I changed the structure!

File: docs/setup/Connecting.md (hunk 7 above)
Comment (Clockworkx) [reply docs/setup/Connecting.md:44]: addressed in commit.

File: docs/setup/Connecting.md (hunk 10 above)
Comment (Clockworkx) [reply docs/setup/Connecting.md:49]: addressed in commit.

File: docs/setup/running.md (hunk 8 above)
Comment (Clockworkx) [reply docs/setup/running.md]: addressed

File: docs/setup/prerequisites.md (hunk 12 above)
Comment (Clockworkx) [reply docs/setup/prerequisites.md:23]: free for students tho, so I would leave it in, also other more experienced devs might be happy to hear they can use their favorite IDE without problems,

**Pull request merged**